mw-webapp/src/component/header/Header.tsx

```diff
@@ -25,26 +23,20 @@ export const Header = () => {
 
   return (
     <div className={styles.header}>
-      <h1 className={styles.title}>
-        {LOGO_TEXT.toUpperCase()}
-      </h1>
-      {user && <h2 className={styles.title}>
-        Hello,
-        {" "}
-        {user.displayName}
-        !
-      </h2>}
-      <div className={styles.blockButton}>
-        <Button
-          onClick={user ? logOut : logIn}
-          value={user ? BUTTON_LOG_OUT_VALUE : BUTTON_LOG_IN_VALUE}
+      <Title
```

Comment:
Maybe it would be better if the title was additionally designed as a link leading to the main page?

Reply:
As I understand this question about design, not about optimization requets?